ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/HomeScreenMapContainerFragment.kt

```diff
@@ -128,6 +130,29 @@ class HomeScreenMapContainerFragment : AbstractMapContainerFragment() {
     super.onViewCreated(view, savedInstanceState)
     setupMenuFab()
     setupBottomLoiCards()
+    lifecycleScope.launch {
+      mapContainerViewModel.activeSurvey
+        .combine(mapContainerViewModel.lois) { x, y -> Pair(x, y) }
+        .collect {
+          val survey = it.first
+          val lois = it.second
+          survey?.let {
+            val suggest =
+              it.jobs.any {
+                it.strategy != Job.DataCollectionStrategy.PREDEFINED &&
+                  it.strategy != Job.DataCollectionStrategy.UNKNOWN
+              }
+            val readonly = lois.isEmpty()
+            val msg =
+              when {
+                suggest -> resources.getString(R.string.suggest_data_collection_hint)
+                readonly -> resources.getString(R.string.read_only_data_collection_hint)
+                else -> resources.getString(R.string.predefined_data_collection_hint)
+              }
+            Snackbar.make(binding.root, msg, Snackbar.LENGTH_LONG).show()
```

Comment:
Per designs the Snackbar should stick around until dismissed. If not this in this PR, please add a TODO(bug) to add that.

Reply:
I somehow missed that, done!